bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/internal/RuleImpl.java

```diff
@@ -45,92 +46,88 @@
 public class RuleImpl implements Rule {
 
-    @NonNullByDefault({})
-    private List<Trigger> triggers;
-    @NonNullByDefault({})
-    private List<Condition> conditions;
-    @NonNullByDefault({})
-    private List<Action> actions;
-    @NonNullByDefault({})
-    private Configuration configuration;
-    @NonNullByDefault({})
-    private List<ConfigDescriptionParameter> configDescriptions;
-    @Nullable
-    private String templateUID;
-    @NonNullByDefault({})
-    private String uid;
-    @Nullable
-    private String name;
-    @NonNullByDefault({})
-    private Set<String> tags;
-    @NonNullByDefault({})
-    private Visibility visibility;
-    @Nullable
-    private String description;
+    protected @NonNullByDefault({}) List<Trigger> triggers;
+    protected @NonNullByDefault({}) List<Condition> conditions;
+    protected @NonNullByDefault({}) List<Action> actions;
+    protected @NonNullByDefault({}) Configuration configuration;
+    protected @NonNullByDefault({}) List<ConfigDescriptionParameter> configDescriptions;
+    protected @Nullable String templateUID;
+    protected @NonNullByDefault({}) String uid;
+    protected @Nullable String name;
+    protected @NonNullByDefault({}) Set<String> tags;
+    protected @NonNullByDefault({}) Visibility visibility;
+    protected @Nullable String description;
 
     private transient volatile RuleStatusInfo status = new RuleStatusInfo(RuleStatus.UNINITIALIZED,
             RuleStatusDetail.NONE);
 
     /**
-     * Package protected default constructor to allow reflective instantiation.
-     *
-     * !!! DO NOT REMOVE - Gson needs it !!!
-     */
-    RuleImpl() {
-    }
-
-    /**
-     * Constructor for creating an empty {@link RuleImpl} with a specified rule identifier.
-     * When {@code null} is passed for the {@code uid} parameter, the {@link RuleImpl}'s identifier will
+     * Constructor for creating an empty {@link Rule} with a specified rule identifier.
+     * When {@code null} is passed for the {@code uid} parameter, the {@link Rule}'s identifier will
      * be randomly generated.
      *
      * @param uid the rule's identifier, or {@code null} if a random identifier should be generated.
      */
     public RuleImpl(@Nullable String uid) {
-        this(uid, null, null, null, null, null, null, null);
+        this(uid, null, null, null, null, null, null, null, null, null, null);
     }
 
     /**
-     * Utility constructor for creating a {@link RuleImpl} from a set of modules, or from a template.
-     * When {@code null} is passed for the {@code uid} parameter, the {@link RuleImpl}'s identifier will be randomly
+     * Utility constructor for creating a {@link Rule} from a set of modules, or from a template.
+     * When {@code null} is passed for the {@code uid} parameter, the {@link Rule}'s identifier will be randomly
+     * generated.
+     * }
+     * 
+     * /**
+     * Utility constructor for creating a {@link Rule} from a set of modules, or from a template.
```

Comment:
something went wrong editing this comment

Reply:
No, it has been done wrong in a previous PR. There has been wrong indent rules for JavaDoc comment on Photon. Do you remember?
As now the "old" formatter rules has been applied again (the IDE formatter setup has been fixed) the added whitespaces are removed again.
What should be done is to remove the non necessary line breaks (that also exists in the old base).

Reply:
I was talking about the `Utility constructor for creating...` sentence which appears twice in the comment now.

Reply:
I see. Good catch. Will fix it...